package.json

```diff
@@ -14,6 +14,7 @@
         "url": "git+https://github.com/Dirnei/node-red-contrib-zigbee2mqtt-devices"
     },
     "node-red": {
+        "version": ">=2.0.0",
```

Comment:
@Dirnei what do you think about restricting the project to Node-RED 2.0 - as I personally don't test it with 1.3 and 1.3 is EOL in July anyway.
But: In theory, it still works with 1.3.

Reply:
I changed it to 1.2.2 for now as it is my current deployment. Shouldn't be a problem as we don't use any new features